cedar-policy-validator/src/schema.rs

```diff
@@ -674,16 +768,91 @@ impl TryInto<ValidatorSchema> for NamespaceDefinitionWithActionAttributes<RawNam
     }
 }
 
+/// Get a `ValidatorSchemaFragment` describing the items that implicitly exist
+/// in the `__cedar` namespace.
+fn cedar_fragment(extensions: Extensions<'_>) -> ValidatorSchemaFragment<ConditionalName> {
+    // PANIC SAFETY: these are valid `Id`s
+    #[allow(clippy::unwrap_used)]
+    let mut common_types = HashMap::from_iter([
+        (
+            Id::from_str("Bool").unwrap(),
```

Comment:
That constructor is currently private to Core.  Should we make it public from Core?

Reply:
For hotness, this function is called exactly once per `ValidatorSchema` that is constructed.  Which is probably not hot enough to matter.  There are a lot of other things going on when constructing a `ValidatorSchema` that the performance here will probably be lost in the noise.  Maybe if we benchmarked specifically constructing an empty schema this would be a noticeable component of the runtime